incubator/etcd/values.yaml

```diff
@@ -6,13 +6,24 @@
 Name: etcd
 PeerPort: 2380
 ClientPort: 2379
+# ProxyPort: 23790
 Component: "etcd"
+PeerTLS: true
+# InitialSize should be set only at INSTALL, after the cluster is live, don't change this setting.
+InitialSize: 3
```

Comment:
Asking for problems by having a setting that shouldn't be changed. As before, why can't we just stick with the one var for current size?

Reply:
yeah, why I removed it.